pymeasure/adapters/adapter.py

```diff
@@ -33,48 +37,135 @@ class Adapter:
 
     This class should only be inherited from.
 
-    :param preprocess_reply: optional callable used to preprocess strings
-        received from the instrument. The callable returns the processed string.
+    :param preprocess_reply: (deprecated) optional callable used to preprocess
```

Comment:
I agree with deprecating it, I am not sure why preprocessing a reply is adapter dependent (and not instrument dependent).
I see two cases:
- my instrument for some reason requires some processing of his messages, in such case, I would implement it in the `read` method or similar.
- some command of my instrument requires a reply preprocessing, in this case I would use the parameter in the `Instrument.control/measurement` as appropriate.

So I agree to deprecate it, but I would not add it to the `Instrument` parameters.

Reply:
You're right. From looking at the whole Rework, I forgot to look, whether all functionality of the moved `values` is still necessary.

I see just one use case for an Instrument.preprocess_reply: Reading values (in the control) is different from reading in methods.
This is, I guess, more a hypothetical point and in this case you could add the preprocess to every control manually.

Reply:
I removed the preprocess_reply parameter (and attribute) of `Instrument` and adjusted the docstrings accordingly.

Reply:
I'm pretty confident that this can be closed.